modules/lti/src/main/resources/tools/manage/js/lti.manage.js

```diff
@@ -1112,6 +1112,22 @@ function personalEventEditable(id) {
             '  <i class="fa fa-times-circle"></i></button></div>';
   return str;
 }
+function getTooltip(details){
+    var status = getStatus(details);
+
+    if (status == "Processing"){
+        return "Being processed for publication";
+    }
+    else if (status == "Unwanted"){
+        return "No lecture/consent OR removal requested";
```

Comment:
"No event, no consent provided, or recording was published and later retracted."